ccmlib/cmds/cluster_cmds.py

```diff
@@ -345,6 +346,22 @@ def run(self):
         self.cluster.show(self.options.verbose)
 
 
+class ClusterShowlogsCmd(Cmd):
+    descr_text = "Show logs of nodes in this claster. If no nodes are specified, logs of all nodes will be shown.\
+                 By default multitail is used. If you need to alter the command or options, change CCM_MULTITAIL_CMD."
+
+    usage = "usage: ccm showlogs [node1 node2 ...]"
+
+    def validate(self, parser, options, args):
+        Cmd.validate(self, parser, options, args, load_cluster=True)
+
+    def run(self):
+        shower = os.getenv("CCM_MULTITAIL_CMD", 'multitail').split()[0]
```

Comment:
Can you remove all of the `CCM_MULTITAIL_CMD` env based configuration? I don't think we need to support multiple options.

Reply:
Thanks for your comment, however original issue #372 requests this feature and I believe that this feature is useful when the users do not have multitail in their systems or they want to flexibly change the arrangement of mini-panes (vertical split, horizontal split, etc).

Reply:
Hmm, you're right, Jim did ask for that. Nevermind then.